plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -231,6 +233,10 @@ protected void verifyConcurrentUpdateFailurePermissible(Exception e)
     protected void verifyConcurrentAddColumnFailurePermissible(Exception e)
     {
         assertThat(e)
+                .hasMessageMatching("Failed to add column col\\d")
```

Comment:
Prefer `hasMessage`

Reply:
this error message is varying because it contains the ID of the thread itself, so we can't use `hasMessage()` here